### PR TITLE
Add documentation for the excludeResourcePrefixes

### PR DESCRIPTION
**Pull request by tenzen-y**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
There is a lack of documentation for the `resources.excludeResourcePrefixes`: #2267.
Also, I added the default parameter to manifests.

#### Which issue(s) this PR fixes:

Part-of #2266

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md [hunk 1]
```diff
@@ -384,3 +384,21 @@ following command:
 ```shell
 kubectl apply -f team-a-cq.yaml -f team-b-cq.yaml -f shared-cq.yaml
 ```
+
+## Exclude arbitrary resources in the quota management 
+By default, administrators must specify all resources used by any Pods in the ClusterQueues `.spec.resourceGroups[*]`.
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/administer_cluster_quotas.md:389]:
```suggestion
By default, administrators must specify all resources required by Pods in the ClusterQueues `.spec.resourceGroups[*]`.
```

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md [hunk 2]
```diff
@@ -384,3 +384,21 @@ following command:
...
+If you want to exclude some resources for the specific devices in the ClusterQueues management,
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/administer_cluster_quotas.md:390]:
```suggestion
If you want to exclude some resources in the ClusterQueues quota management and admission process, 
```

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md [hunk 3]
```diff
@@ -384,3 +384,21 @@ following command:
 ```shell
 kubectl apply -f team-a-cq.yaml -f team-b-cq.yaml -f shared-cq.yaml
 ```
+
+## Exclude arbitrary resources in the quota management 
+By default, administrators must specify all resources used by any Pods in the ClusterQueues `.spec.resourceGroups[*]`.
+If you want to exclude some resources for the specific devices in the ClusterQueues management, 
+you can specify the resource prefixes in the Kueue configuration as a cluster-level setting.
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/administer_cluster_quotas.md:391]:
```suggestion
you can specify the resource prefixes in the Kueue Configuration as a cluster-level setting.
```

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md [hunk 4]
```diff
@@ -384,3 +384,21 @@ following command:
...
+
+Follow the instructions described
+[here](/docs/installation#install-a-custom-configured-released-version) to
+install a release version by extending the configuration with the following
+fields:
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/administer_cluster_quotas.md:396]:
```suggestion
Follow the [installation instructions for using a custom configuration](/docs/installation#install-a-custom-configured-released-version)
and extend the configuration with fields similar to the following:
```

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md [hunk 5]
```diff
@@ -384,3 +384,21 @@ following command:
...
+
+```yaml
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/administer_cluster_quotas.md:398]:
```suggestion
```yaml
apiVersion: config.kueue.x-k8s.io/v1beta1
kind: Configuration
```

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md (hunk 1 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/administer_cluster_quotas.md]: Done.

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md (hunk 2 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/administer_cluster_quotas.md]: Done.

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md (hunk 3 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/administer_cluster_quotas.md]: Done.

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md (hunk 4 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/administer_cluster_quotas.md]: Done.

File: site/content/en/docs/tasks/manage/administer_cluster_quotas.md (hunk 5 above)
Comment (tenzen-y) [reply site/content/en/docs/tasks/manage/administer_cluster_quotas.md:396]: Done.

**Comment by tenzen-y** [comment]
@alculquicondor I addressed all comments. PTAL, thanks.

**Pull request merged**